packages/javascript/bh-shared-ui/src/components/ExploreTable/useExploreTableRowsAndColumns.tsx

```
@@ -0,0 +1,195 @@
+    const handleSort = useCallback(
+        (sortByColumn: keyof MungedTableRowWithId) => {
+            if (sortByColumn) {
+                if (sortBy === sortByColumn) {
+                    switch (sortOrder) {
+                        case 'desc':
+                            setSortOrder('asc');
+                            break;
+                        case 'asc':
+                            setSortOrder('desc');
+                            break;
+                        default:
```

Biome (1.9.4)

[error] The default clause should be the last switch clause.

The following case clause is here:

Regardless its position, the default clause is always executed when there is no match. To avoid confusion, the default clause should be the last switch clause.

(lint/suspicious/useDefaultSwitchClauseLast)